Sources/CvSelectionGroupAI.cpp

```diff
@@ -1270,17 +1246,14 @@ CvUnit* CvSelectionGroupAI::AI_ejectBestPropertyManipulator(CvCity* pTargetCity)
 
 CvUnit* CvSelectionGroupAI::AI_findBestDefender(CvPlot* pDefendPlot, bool allowAllDefenders, bool bConsiderPropertyValues)
 {
-	CLLNode<IDInfo>* pEntityNode;
-	CvUnit* pLoopUnit;
-
-	pEntityNode = headUnitNode();
+	CLLNode<IDInfo>* pEntityNode = headUnitNode();
 	
 	CvUnit* pBestUnit = NULL;
 	int iBestUnitValue = 0;
 
 	while (pEntityNode != NULL)
 	{
-		pLoopUnit = ::getUnit(pEntityNode->m_data);
```

Comment:
Okay I'm not commenting for loop again after this: for loop!

Reply:
Done or will be done.